corehq/apps/analytics/views.py

```diff
@@ -11,3 +18,27 @@ def post(self, request, *args, **kwargs):
         meta = get_meta(request)
         track_clicked_deploy_on_hubspot.delay(request.couch_user, request.COOKIES, meta)
         return HttpResponse()
+
+
+class GreenHouseCandidate(View):
```

Comment:
can we rename this to GreenhouseCandidateView, so if its referenced elsewhere we will be able to know that its a view. It currently sounds  like a model. Otherwise this looks good and sorry for the slow review